core/domain/html_validation_service.py

```diff
@@ -851,20 +855,117 @@ def add_dimensions_to_image_tags(exp_id, html_string):
     return python_utils.UNICODE(soup).replace('<br/>', '<br>')
 
 
-def get_filename_with_dimensions(old_filename, exp_id):
+def add_dimensions_to_image_tags_inside_tabs_and_collapsible_blocks(
```

Comment:
Same thing about not passing context here.

Reply:
Passing file_system instead of exp_id. Done.